universal-gateway/examples/n8n.mdx

```diff
@@ -22,22 +21,64 @@ We want your feedback!
 Sign up for early access to [ngrok's AI Gateway](https://ngrok.ai) and our team will reach out to learn more about your needs.
 </Tip>
 
-## 1. Reserve a domain
+## Step 1: Reserve a domain
 
 <ReserveDomain domain="https://your-n8n-instance.ngrok.app" />
```

Comment:
I noticed that this component doesn't actually seem to read `domain`. I did see in the snippet that it's trying to, but on the page this renders not with the domain we specify here, but with the default. Not sure what's going on there.

<img width="1242" height="480" alt="Image" src="https://github.com/user-attachments/assets/e37b4486-4b27-4408-acf8-3f0b47bee2a9" />

Reply:
Thanks for catching that! I pushed up a fix to this PR